spec.md

```diff
@@ -210,11 +210,39 @@ help intermediate gateways determine how to route the events.
 * Constraints:
   * REQUIRED
 * Examples
-    * https://github.com/cloudevents/spec/pull/123
-    * /cloudevents/spec/pull/123
-    * urn:event:from:myapi/resourse/123
+    * https://github.com/cloudevents/spec/pull
+    * /cloudevents/spec/pull
+    * urn:event:from:myapi/resource
     * mailto:cncf-wg-serverless@lists.cncf.io
 
+### subject
+* Type: `String`
+* Description: This describes the subject of the event in the context of the
+  event producer (identified by `source`). In publish-subscribe scenarios,
+  a subscriber will typically subscribe to events emitted by a `source`,
+  but the `source` identifier alone might not be sufficient as a qualifier for
+  any specific event if the `source` context has internal sub-structure.
+
+  Identifying the subject of the event in context metadata (opposed to only in
+  the `data` payload) is particularly helpful in generic subscription filtering
+  scenarios where middleware is unable to interpret the `data` content. In the
+  above example, the subscriber might only be interested in blobs with names
+  ending with '.jpg' or '.jpeg' and the subject attribute allows for 
```

Comment:
s/subject/`subject`/

Reply:
I like this paragraph. Would it be helpful to add something about how putting this info into the "source" URI is just as hard to parse/filter-on ? And that's why we creating this property. Before this property I think people were pushed towards merging it into 'source' - if they wanted it outside of 'data'.  Or, if this is too much text for the spec, perhaps in the primer?